### PR TITLE
Update both Ignite UI for Angular and Angular to version 7

### LINKED ISSUE #767: Upgrade igniteui-angular to 7.0.0
### Description  
igniteui-angular 7.0.0 was released, angular-samples should be upgraded with the latest version

### PR DESCRIPTION
**Pull request by zdrawku**
Closes #767

### CODE REVIEW

**Comment by hanastasov** [comment]
Found a couple of issues, mostly minor, I am going to log issues about those in igniteui-angular-samples and and igniteui-angular repositories.

File: src/app/combo/cascading-combos/cascading-combos.component.ts [hunk 1]
```diff
@@ -18,8 +18,8 @@ export class CascadingCombos {
 
     @ViewChild("inputGroupCountry", { read: IgxInputGroupComponent}) public inputGroupCountry: IgxInputGroupComponent;
     @ViewChild("inputGroupProvince", { read: IgxInputGroupComponent}) public inputGroupProvince: IgxInputGroupComponent;
-    @ViewChild("inputCountry") public inputCountry: HTMLInputElement;
-    @ViewChild("inputProvince") public inputProvince: HTMLInputElement;
+    // @ViewChild("inputCountry") public inputCountry: HTMLInputElement;
```
Comment (bkulov) [inline src/app/combo/cascading-combos/cascading-combos.component.ts]: Do you need this commented code?

File: src/app/expansion-panel/expansion-sample-5/expansion-sample-5.component.ts [hunk 2]
```diff
@@ -41,9 +41,9 @@ export class ExpansionPanelSample5Component {
     public onInteraction(event) {
         const expandedPanels = this.accordion.filter((panel) => !panel.collapsed);
         expandedPanels.forEach((expandedPanel) => {
-            if (expandedPanel.elementRef !==  event.event.currentTarget) {
-                expandedPanel.collapse();
-            }
+            // if (expandedPanel.elementRef !==  event.event.currentTarget) {
```
Comment (bkulov) [inline src/app/expansion-panel/expansion-sample-5/expansion-sample-5.component.ts]: Do you need this commented code?

File: src/app/combo/cascading-combos/cascading-combos.component.ts (hunk 1 above)
Comment (zdrawku) [reply src/app/combo/cascading-combos/cascading-combos.component.ts]: Yes, I have logged separate [issue](https://github.com/IgniteUI/igniteui-angular-samples/issues/770) for that changes to @IvayloG and @ViktorSlavov in order not to delay this PR

File: src/app/expansion-panel/expansion-sample-5/expansion-sample-5.component.ts (hunk 2 above)
Comment (zdrawku) [reply src/app/expansion-panel/expansion-sample-5/expansion-sample-5.component.ts]: Same my other comment

**Comment by zdrawku** [comment]
New [PR ](https://github.com/IgniteUI/igniteui-angular-samples/pull/780) was created

**Pull request closed without merging**